jf_agent/git/bitbucket_server.py

```diff
@@ -289,11 +299,13 @@ def _normalize_pr_repo(repo, redact_names_and_urls):
 
 
 def get_pull_requests(
-    client, api_repos, strip_text_content, server_git_instance_info, redact_names_and_urls
+    client, api_repos, strip_text_content, server_git_instance_info, redact_names_and_urls, verbose
 ):
     for i, api_repo in enumerate(api_repos, start=1):
         with agent_logging.log_loop_iters(logger, 'repo for pull requests', i, 1):
             repo = api_repo.get()
+            if verbose:
+                print(f"Beginning download of PRs for repo {repo}")
```

Comment:
Maybe for consistency, match this with the other verbose terms?  
Apply `Beginning download` to `Getting list of commits` to make `Beginning download of commits for repo {repo}`